libmscore/edit.cpp

```diff
@@ -1810,7 +1811,8 @@ void Score::deleteItem(Element* el)
         undoAddCR(rest, segment->measure(), segment->tick());
     }
     // fall through
-
+    case ElementType::MMREST:
+    // fall through
```

Comment:
comment not needed, nit to prevent compiler warnings at least, as there is no code. Different to the same comment above. Shouldn't harm though either